assets/collage.css

```diff
@@ -106,28 +106,35 @@
   }
 }
 
+.collage__item--video:focus-within {
+  box-shadow: 0 0 0 .3rem rgb(var(--color-background)),0 0 .5rem .4rem rgba(var(--color-foreground),.3);
+  outline-offset: 0.3rem;
+  outline: .2rem solid rgba(var(--color-foreground),.5);
+}
+
 .collage-card {
```

Comment:
Would love to eventually refactor these to use the base `card` class for most of this stuff to reduce the amount of CSS.

Reply:
Created an issue here: https://github.com/Shopify/dawn/issues/1136